jsonschema2pojo-integration-tests/src/test/java/org/jsonschema2pojo/integration/config/FastJson1IT.java

```diff
@@ -0,0 +1,115 @@
+ * limitations under the License.
+ */
+
+package org.jsonschema2pojo.integration.config;
+
+import com.alibaba.fastjson.JSON;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import org.apache.commons.io.IOUtils;
+import org.jsonschema2pojo.integration.util.Jsonschema2PojoRule;
+import org.junit.Rule;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.Map;
+
+import static org.hamcrest.Matchers.*;
+import static org.jsonschema2pojo.integration.util.CodeGenerationHelper.config;
+import static org.jsonschema2pojo.integration.util.FileSearchMatcher.containsText;
+import static org.jsonschema2pojo.integration.util.JsonAssert.assertEqualsJson;
+import static org.junit.Assert.assertThat;
+
+public class FastJson1IT {
+    @Rule
+    public Jsonschema2PojoRule schemaRule = new Jsonschema2PojoRule();
+
+    @Test
+    @SuppressWarnings({"rawtypes", "unchecked"})
+    public void annotationStyleFastJsonProducesFastJsonAnnotations() throws ClassNotFoundException, SecurityException, NoSuchMethodException {
+
+        Class generatedType = schemaRule.generateAndCompile("/json/examples/torrent.json", "com.example",
+                        config("annotationStyle", "fastjson",
+                                "propertyWordDelimiters", "_",
+                                "sourceType", "json"))
+                .loadClass("com.example.Torrent");
+
+        assertThat(schemaRule.getGenerateDir(), not(containsText("jakarta.json.bind.annotation")));
+        assertThat(schemaRule.getGenerateDir(), not(containsText("javax.json.bind.annotation")));
+        assertThat(schemaRule.getGenerateDir(), not(containsText("org.codehaus.jackson")));
+        assertThat(schemaRule.getGenerateDir(), not(containsText("com.fasterxml.jackson")));
+        assertThat(schemaRule.getGenerateDir(), not(containsText("com.google.gson")));
+        assertThat(schemaRule.getGenerateDir(), not(containsText("@SerializedName")));
+        assertThat(schemaRule.getGenerateDir(), not(containsText("com.squareup.moshi")));
+        assertThat(schemaRule.getGenerateDir(), not(containsText("@com.squareup.moshi.Json")));
+        assertThat(schemaRule.getGenerateDir(), containsText("com.alibaba.fastjson"));
+        assertThat(schemaRule.getGenerateDir(), containsText("@JSONField"));
+
+        Method getter = generatedType.getMethod("getBuild");
+
+        assertThat(generatedType.getAnnotation(JsonPropertyOrder.class), is(nullValue()));
+        assertThat(generatedType.getAnnotation(JsonInclude.class), is(nullValue()));
+        assertThat(getter.getAnnotation(JsonProperty.class), is(nullValue()));
+    }
+
+    @Test
+    public void annotationStyleFastJsonMakesTypesThatWorkWithFastJson() throws ClassNotFoundException, SecurityException, IOException {
```

Comment:
Nit: `ClassNotFoundException, SecurityException, IOException` could be replaced with `ReflectiveOperationException, IOException`